training/model/Containerfile

```diff
@@ -1,3 +1,4 @@
 FROM registry.access.redhat.com/ubi9/ubi
+RUN if [ -z $HF_TOKEN ]; then; exit 1; fi;
```

Comment:
You should print an error message to the user telling them that this is required.